### ISSUE TITLE
64位ocr问题

### ISSUE THREAD

**Issue by grweiz** [WallBreaker2/op#166]
使用ocr的时候，打印这么一堆log后识别不到任何东西
OcrWrapper::OcrWrapper()
In PP-OCRv3, rec_image_shape parameter defaults to '3, 48, 320',if you are using recognition model with PP-OCRv2 or an older version, please set --rec_image_shape='3,32,320
WARNING: Logging before InitGoogleLogging() is written to STDERR
I0313 15:13:03.123385 21852 analysis_predictor.cc:1626] MKLDNN is enabled
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
I0313 15:13:03.141937 21852 executor.cc:187] Old Executor is Running.
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [mkldnn_placement_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [matmul_v2_scale_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_map_matmul_v2_to_mul_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_map_matmul_v2_to_matmul_pass]e[0m
e[32m--- Running IR pass [matmul_scale_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_map_matmul_to_mul_pass]e[0m
e[32m--- Running IR pass [fc_fuse_pass]e[0m
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0313 15:13:03.269840 21852 fuse_pass_base.cc:59] ---  detected 33 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_eltwiseadd_bn_fuse_pass]e[0m
I0313 15:13:03.295215 21852 fuse_pass_base.cc:59] ---  detected 1 subgraphs
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [constant_folding_pass]e[0m
e[32m--- Running IR pass [squeeze2_transpose2_onednn_fuse_pass]e[0m
e[37m--- fused 0 squeeze2 with transpose2e[0m
e[32m--- Running IR pass [depthwise_conv_mkldnn_pass]e[0m
I0313 15:13:03.315711 21852 fuse_pass_base.cc:59] ---  detected 15 subgraphs
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0313 15:13:03.335232 21852 fuse_pass_base.cc:59] ---  detected 15 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_affine_channel_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bias_mkldnn_fuse_pass]e[0m
I0313 15:13:03.361583 21852 fuse_pass_base.cc:59] ---  detected 16 subgraphs
e[37m---    fused 16 conv2d with elementwise_add as biase[0m
e[32m--- Running IR pass [conv_transpose_bias_mkldnn_fuse_pass]e[0m
I0313 15:13:03.370368 21852 fuse_pass_base.cc:59] ---  detected 2 subgraphs
e[37m---    fused 2 conv2d_transpose with elementwise_add as biase[0m
e[32m--- Running IR pass [conv_elementwise_add_mkldnn_fuse_pass]e[0m
e[37m---    Fused 10 conv (as y) + elementwise_add patternse[0m
I0313 15:13:03.434962 21852 fuse_pass_base.cc:59] ---  detected 10 subgraphs
e[32m--- Running IR pass [conv_activation_mkldnn_fuse_pass]e[0m
I0313 15:13:03.441794 21852 fuse_pass_base.cc:59] ---  detected 8 subgraphs
e[37m---    fused 8 fused_conv2d with hard_sigmoid activatione[0m
I0313 15:13:03.446673 21852 fuse_pass_base.cc:59] ---  detected 20 subgraphs
e[37m---    fused 20 fused_conv2d with hard_swish activatione[0m
I0313 15:13:03.454481 21852 fuse_pass_base.cc:59] ---  detected 21 subgraphs
e[37m---    fused 21 fused_conv2d with relu activatione[0m
e[32m--- Running IR pass [scale_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [reshape_transpose_matmul_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [matmul_transpose_reshape_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [matmul_elementwise_add_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [matmul_activation_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [fc_mkldnn_pass]e[0m
e[32m--- Running IR pass [fc_act_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [self_attention_fuse_pass]e[0m
W0313 15:13:03.473026 21852 self_attention_fuse_pass.cc:53] No-avx512 or MKL supported!
e[32m--- Running IR pass [batch_norm_act_fuse_pass]e[0m
e[32m--- Running IR pass [softplus_activation_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [shuffle_channel_mkldnn_detect_pass]e[0m
e[32m--- Running IR pass [elementwise_act_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [operator_scale_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [operator_unsqueeze2_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [operator_reshape2_onednn_fuse_pass]e[0m
e[1me[35m--- Running analysis [save_optimized_model_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0313 15:13:03.486689 21852 memory_optimize_pass.cc:118] The persistable params in main graph are : 2.18927MB
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : hardswish_16.tmp_0  size: 1920
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : tmp_8  size: 384
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : hardswish_15.tmp_0  size: 1920
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : tmp_5  size: 384
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : tmp_10  size: 384
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : tmp_9  size: 384
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : tmp_1  size: 384
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : elementwise_add_1  size: 64
I0313 15:13:03.489616 21852 memory_optimize_pass.cc:246] Cluster name : x  size: 12
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0313 15:13:03.562817 21852 analysis_predictor.cc:1838] ======= optimize end =======
I0313 15:13:03.562817 21852 naive_executor.cc:200] ---  skip [feed], feed -> x
I0313 15:13:03.564769 21852 naive_executor.cc:200] ---  skip [sigmoid_0.tmp_0], fetch -> fetch
In PP-OCRv3, default rec_img_h is 48,if you use other model, you should set the param rec_img_h=32
I0313 15:13:03.584288 21852 analysis_predictor.cc:1626] MKLDNN is enabled
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [mkldnn_placement_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [matmul_v2_scale_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_map_matmul_v2_to_mul_pass]e[0m
I0313 15:13:03.669201 21852 fuse_pass_base.cc:59] ---  detected 9 subgraphs
e[32m--- Running IR pass [gpu_cpu_map_matmul_v2_to_matmul_pass]e[0m
I0313 15:13:03.670177 21852 fuse_pass_base.cc:59] ---  detected 4 subgraphs
e[32m--- Running IR pass [matmul_scale_fuse_pass]e[0m
e[32m--- Running IR pass [gpu_cpu_map_matmul_to_mul_pass]e[0m
e[32m--- Running IR pass [fc_fuse_pass]e[0m
I0313 15:13:03.677985 21852 fuse_pass_base.cc:59] ---  detected 9 subgraphs
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0313 15:13:03.740449 21852 fuse_pass_base.cc:59] ---  detected 19 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [constant_folding_pass]e[0m
e[32m--- Running IR pass [squeeze2_transpose2_onednn_fuse_pass]e[0m
I0313 15:13:03.762897 21852 fuse_pass_base.cc:59] ---  detected 1 subgraphs
e[37m--- fused 1 squeeze2 with transpose2e[0m
e[32m--- Running IR pass [depthwise_conv_mkldnn_pass]e[0m
I0313 15:13:03.762897 21852 fuse_pass_base.cc:59] ---  detected 13 subgraphs
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0313 15:13:03.782418 21852 fuse_pass_base.cc:59] ---  detected 13 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_affine_channel_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bias_mkldnn_fuse_pass]e[0m
I0313 15:13:03.790225 21852 fuse_pass_base.cc:59] ---  detected 4 subgraphs
e[37m---    fused 4 conv2d with elementwise_add as biase[0m
e[32m--- Running IR pass [conv_transpose_bias_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_elementwise_add_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_activation_mkldnn_fuse_pass]e[0m
I0313 15:13:03.808768 21852 fuse_pass_base.cc:59] ---  detected 2 subgraphs
e[37m---    fused 2 fused_conv2d with hard_sigmoid activatione[0m
I0313 15:13:03.814625 21852 fuse_pass_base.cc:59] ---  detected 27 subgraphs
e[37m---    fused 27 fused_conv2d with hard_swish activatione[0m
I0313 15:13:03.816577 21852 fuse_pass_base.cc:59] ---  detected 2 subgraphs
e[37m---    fused 2 fused_conv2d with relu activatione[0m
W0313 15:13:03.819505 21852 op_compat_sensible_pass.cc:220]  Attribute(beta) of Op(swish) is not defined in opProto or is in extra set!The compatable check for this attribute is not use. Please remove it from the precondition of pass: conv_activation_mkldnn_fuse_pass
I0313 15:13:03.820482 21852 fuse_pass_base.cc:59] ---  detected 5 subgraphs
e[37m---    fused 5 fused_conv2d with swish activatione[0m
e[32m--- Running IR pass [scale_matmul_fuse_pass]e[0m
I0313 15:13:03.821457 21852 fuse_pass_base.cc:59] ---  detected 2 subgraphs
e[37m---    fused 2 scale with matmule[0m
e[32m--- Running IR pass [reshape_transpose_matmul_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [matmul_transpose_reshape_mkldnn_fuse_pass]e[0m
I0313 15:13:03.834146 21852 fuse_pass_base.cc:59] ---  detected 2 subgraphs
e[37m---    fused 2 matmul + transpose + reshape patternse[0m
e[32m--- Running IR pass [matmul_elementwise_add_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [matmul_activation_mkldnn_fuse_pass]e[0m
e[32m--- Running IR pass [fc_mkldnn_pass]e[0m
I0313 15:13:03.842929 21852 fuse_pass_base.cc:59] ---  detected 9 subgraphs
e[37m---    enabled FC MKL-DNN for 9 fc ops e[0m
e[32m--- Running IR pass [fc_act_mkldnn_fuse_pass]e[0m
I0313 15:13:03.845857 21852 fuse_pass_base.cc:59] ---  detected 2 subgraphs
e[37m---    fused 2 fc with swish activatione[0m
e[32m--- Running IR pass [self_attention_fuse_pass]e[0m
W0313 15:13:03.845857 21852 self_attention_fuse_pass.cc:53] No-avx512 or MKL supported!
e[32m--- Running IR pass [batch_norm_act_fuse_pass]e[0m
e[32m--- Running IR pass [softplus_activation_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [shuffle_channel_mkldnn_detect_pass]e[0m
e[32m--- Running IR pass [elementwise_act_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [operator_scale_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [operator_unsqueeze2_onednn_fuse_pass]e[0m
e[32m--- Running IR pass [operator_reshape2_onednn_fuse_pass]e[0m
e[1me[35m--- Running analysis [save_optimized_model_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0313 15:13:03.857569 21852 memory_optimize_pass.cc:118] The persistable params in main graph are : 10.0455MB
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : transpose_14.tmp_1  size: 0
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : linear_43.tmp_1  size: 26500
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : x  size: 576
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : batch_norm_40.tmp_4  size: 6144
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : batch_norm_50.tmp_4  size: 3072
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : relu_3.tmp_0  size: 512
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : transpose_9.tmp_0  size: 480
I0313 15:13:03.859520 21852 memory_optimize_pass.cc:246] Cluster name : transpose_13.tmp_0_slice_1  size: 480
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0313 15:13:03.916129 21852 analysis_predictor.cc:1838] ======= optimize end =======
I0313 15:13:03.916129 21852 naive_executor.cc:200] ---  skip [feed], feed -> x
I0313 15:13:03.918081 21852 naive_executor.cc:200] ---  skip [softmax_5.tmp_0], fetch -> fetch
I0313 15:13:03.918081 21852 onednn_context.cc:81] oneDNN v3.2.1
The detection visualized image saved in ./output//paddle_ocr_det.bmp
